cmd/access/node_builder/access_node_builder.go

```diff
@@ -1557,6 +1567,18 @@ func (builder *FlowAccessNodeBuilder) Build() (cmd.Node, error) {
 				return nil, fmt.Errorf("event query mode 'compare' is not supported")
 			}
 
+			broadcaster := engine.NewBroadcaster()
+			// create BlockTracker that will track for new blocks (finalized and sealed) and
+			// handles block-related operations.
+			blockTracker, err := subscription.NewBlockTracker(
+				node.State,
+				builder.executionDataConfig.InitialBlockHeight,
```

Comment:
this should be the root block since it's tracking blocks
```suggestion
				builder.FinalizedRootBlock.Header.Height,
```